linera-core/src/notifier.rs

```diff
@@ -1,11 +1,15 @@
 // Copyright (c) Zefchain Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+use std::sync::Arc;
+
 use dashmap::DashMap;
 use linera_base::identifiers::ChainId;
 use tokio::sync::mpsc::{UnboundedReceiver, UnboundedSender};
 use tracing::trace;
 
+use crate::worker;
```

Comment:
I actually think `Notification` should be exposed from the top level of this crate.

Reply:
I agree, but I'd change that in a separate PR: Currently `linera-core` has almost all its types inside public modules.